server/routes/createGoal/createGoalFormValidator.test.ts

```diff
@@ -1,27 +1,17 @@
 import type { CreateGoalForm } from 'forms'
-import validateGoalTitle from '../../validators/goalTitleValidator'
-import goalTargetCompletionDateValidator from '../../validators/goalTargetCompletionDateValidator'
 import validateCreateGoalForm from './createGoalFormValidator'
 
-jest.mock('../../validators/goalTitleValidator')
-jest.mock('../../validators/goalTargetCompletionDateValidator')
-
 describe('createGoalFormValidator', () => {
-  const mockedValidateGoalTitle = validateGoalTitle as jest.MockedFunction<typeof validateGoalTitle>
```

Comment:
These validators call out to other validators. I've got rid of all the mocks and instead let the validator use the real implementations of the "child" validators.

When it comes to the assertion though, the important thing for _this_ unit test of _this_ validator is that the errors array is correctly built with the right field name. The text is largely irrelevant (because the text comes from the "child" validators, and they have their own unit tests that assert the text). This is why we used mocks before, so as to abstract this test from the implementation of the "child" validators (whose text might change over time)
Because I still dont want to couple _these_ tests with the child validator implementations, I've changed the assertion to assert any string:
EG: I've changed this:
```
expect(errors).toEqual([{ href: '#title', text: 'some-title-error' }])
```
to this:
```
expect(errors).toEqual([{ href: '#title', text: expect.any(String) }])
```